packages/sitecore-jss-angular/angular.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
What is the benefit/reason of configuring `sitecore-jss-angular` as an angular library?

Reply:
In short: in order for tests to be correctly compiled and run.
Previously we used karma-typescript to build tests, but it no longer works with Angular v13+. So instead I use @angular-devkit/build-angular, which has a downside - it cannot be executed outside of an Angular project, but it allows to successfully run tests via ng test.
I was also wrong about re-import from sitecore-jss. It works within Angular logic. The only place it fails is scripts, since they run through ts-node and it doesn't mix well with ESM imports.